packages/jest-config/src/__tests__/normalize-test.js

```diff
@@ -827,14 +827,35 @@ describe('preset', () => {
       {},
     );
 
-    expect(options.moduleNameMapper).toEqual([['b', 'b'], ['a', 'a']]);
+    expect(options.moduleNameMapper).toEqual([['a', 'a'], ['b', 'b']]);
     expect(options.modulePathIgnorePatterns).toEqual(['b', 'a']);
     expect(options.setupFiles.sort()).toEqual([
       '/node_modules/a',
       '/node_modules/b',
       '/node_modules/regenerator-runtime/runtime',
     ]);
   });
+
+  test('merges with options and moduleNameMapper preset is overridden by options', () => {
+    /* eslint-disable sort-keys */
+    // prettier-ignore
```

Comment:
I was just thinking that I could alternatively retain the resilience of the test just by changing semantics. Instead of using the object initializer I could create an empty object and then assign each property in separate statements. Please let me know what you would prefer. Thanks!